CHANGELOG.md

```diff
@@ -10,12 +10,48 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Changed
 
+### Fixed
+
+### Removed
+
+## v0.9.0 - 2022-09-12
+
+### Added
+
+- The new `--level` flag can be set on the command line and will override all other levels, even non-default levels
```

Comment:
we should add this (and the timing flag) to the README as well.  I forgot to do that as part of the ticket.

Reply:
[419aa61](https://github.com/sscpac/statick/pull/446/commits/419aa61bd2ec56ed4f98effcc6043f06130a0332)